tests/e2e/framework/kubernetes.go

```diff
@@ -180,7 +188,7 @@ func (k *KubeInfo) Setup() error {
 		}
 
 	}
-	return nil
+	return checkIngress(k.Ingress + ":80")
```

Comment:
@ldemailly is that what you were talking about ?

Reply:
is that doing a tcp connection check ?

Reply:
why in jason’s fork btw?

Reply:
yeah look up. I am doing this Jasonś fork since he is already making improvements.

Reply:
I'd call it checkTCP() instead of checkIngress() unless you want later to make it do ingress specific stuff
(you could use fortio's fhttp.NewClient() + fetch + code check too I guess)